### ISSUE TITLE
Overlapping Keybind F1

### ISSUE THREAD

**Issue by chi27870** [noxsicarius/Epoch-Admin-Tools#318]
Overlapping keybind for default 1.0.6 earplugs and the F1 teleport function within admin tools.

Thanks for the update guys!!!!!!!

**Comment by MollyXVI** [comment]
What?

**Comment by chi27870** [comment]
Epoch 1.0.6 earplugs = F1
Epoch admin tools map teleport = F1

**Comment by noxsicarius** [comment]
Can easily be modified by users in the keybindings files. Those files were created to allow customization for these types of situations.

**Comment by chi27870** [comment]
Of course, I was just making note of it in case someone wanted to put instructions in the installation tutorial, or in case the author wanted to change the binding.
